README.md

```diff
@@ -83,9 +77,9 @@ Cylc is used to encode the entire pipeline from start to finish and relies on th
    
 ### Running the Cylc pipeline locally
 
-When running locally, make sure you have at least Julia 1.9.0 installed with the correct architecture for your local machine. (https://julialang.org/downloads/)
-To use the Cylc pipeline locally, also make sure Docker Desktop client is running in the background. (https://www.docker.com/products/docker-desktop/)
-
+#### Prerequisites
+__Julia:__ When running locally, make sure you have at least Julia 1.9.0 installed with the correct architecture for your local machine. (https://julialang.org/downloads/)
+__Docker Desktop:__ Also make sure Docker Desktop client is running in the background to use the Cylc pipeline locally. (https://www.docker.com/products/docker-desktop/)
```

Comment:
I thought we were not going to suggest using containers to run things locally?

Reply:
The fetchdata docker container works fine both on hpc and locally, so users will need Docker running for that. The issue is with Julia and arm64 in Docker containers: https://github.com/JuliaPy/Conda.jl/issues/228